### PR TITLE
Better exception formatting.

### PR DESCRIPTION
**Pull request by ihodes**
[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/714)

### CODE REVIEW

File: cycledash/__init__.py [hunk 1]
```diff
@@ -15,19 +17,15 @@ def initialize_application():
...
-        app.logger.exception(exception)
-        return 'Internal Server Error', 500
-
-    import logging
-    import sys
-
     stdout_handler = logging.StreamHandler(sys.stdout)
     stdout_handler.setLevel(logging.INFO)
-    formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+    formatter = logging.Formatter("""\
```
Comment (danvk) [inline cycledash/__init__.py]: is this a literal `\`?

**Comment by danvk** [comment]
Worth noting that we lose our ability to grep logs by going to a multiline format, but that's not something we've ever done in the past.

**Comment by ihodes** [comment]
Our logs have actually always been in a multi line format! This just adds one more newline (+ some more info) to make it easier to read & grep.

File: cycledash/__init__.py (hunk 1 above)
Comment (ihodes) [inline cycledash/__init__.py]: No, it escapes the newline,

Comment (danvk) [inline cycledash/__init__.py]: weird, did not know this trick.

**Pull request merged**